app/views/TaxYears.scala.html

```diff
@@ -42,9 +42,9 @@
 }
 
 @viewReturnLink(year: Int) = {
-    <a class="govuk-link" data-journey-click="EstimatesPage:ClickLink:@year" id="viewReturn-link-@year"
+    <a class="govuk-link" data-journey-click="EstimatesPage:ClickLink:@year" id="viewSummary-link-@year"
```

Comment:
its not great that we are mixing naming conventions here - not your fault, I know..
view-summary-link would be nice